config/mapping.json

```diff
@@ -0,0 +1,137 @@
+{
```

Comment:
This is definitely not a blocker for merging, just something we should have an open issue for...